src/DoctrineWriter.php

```diff
@@ -262,47 +287,50 @@ protected function disableLogging()
      */
     protected function reEnableLogging()
     {
-        $config = $this->entityManager->getConnection()->getConfiguration();
+        //TODO: do we need to add support for MongoDB logging?
+        if (!($this->objectManager instanceof \Doctrine\ORM\EntityManager)) return;
```

Comment:
I would simply throw an `Exception` if the `objectManager` property isn't a `Doctrine\ORM\EntityManager`.